### ISSUE TITLE
"Egress https" part docs for 1.0 is still an old version

### ISSUE THREAD

**Issue by FrostyLeaf** [istio/istio#12054]
**Describe the bug**

I'm trying to allow an external service can be accessed by a meshed service, following [egress-https](https://istio.io/blog/2018/egress-https/) part. When I tried to create an `EgressRule` object, I got this error:

```bash
error: unable to recognize "dev-external-svc.yaml": no matches for kind "EgressRule" in version "networking.istio.io/v1alpha3"
```

Then I fixed it by modifying it as a `ServiceEntry` object.

Seems like the `EgressRule` definition is totally removed from `CustomResourceDefinition` list?

```bash
$ kubectl get crd | grep egress
$
```

**Expected behavior**

Update [egress-https](https://istio.io/blog/2018/egress-https/) part docs, to tell us that we should use new objects to replace the old ones.

**Steps to reproduce the bug**

Just follow the "[egress-https](https://istio.io/blog/2018/egress-https/)" part docs for istio 1.0.

**Version**

```bash
$ istioctl version
Version: 1.0.6
GitRevision: 98598f88f6ee9c1e6b3f03b652d8e0e3cd114fa2
User: root@464fc845-2bf8-11e9-b805-0a580a2c0506
Hub: docker.io/istio
GolangVersion: go1.10.4
BuildStatus: Clean

$ kubectl version
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.2", GitCommit:"cff46ab41ff0bb44d8584413b598ad8360ec1def", GitTreeState:"clean", BuildDate:"2019-01-10T23:35:51Z", GoVersion:"go1.11.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.2", GitCommit:"cff46ab41ff0bb44d8584413b598ad8360ec1def", GitTreeState:"clean", BuildDate:"2019-01-10T23:28:14Z", GoVersion:"go1.11.4", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by FrostyLeaf** [comment]
Docs are fixed in 1.1. Closing it now.

**Comment by howardjohn** [comment]
Docs updated
